### PR TITLE
IDEMPIERE-6064: Error when creating a table with a long name through CreateTable process

### PR DESCRIPTION
**Pull request by nmicoud**
https://idempiere.atlassian.net/browse/IDEMPIERE-6064

# Pull Request Checklist

- [X] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [X] My code follows the best practices of this project
- [X] I have performed a self-review of my own code
- [X] My code is easy to understand and review. 
- [X] I have checked my code and corrected any misspellings
- [X] In hard-to-understand areas, I have commented my code.
- [X] My changes generate no new warnings
### Tests
- [X] I have tested the direct scenario that my code is solving
- [ ] I checked all collaterals that can be affected by my changes, and tested other potential affected scenarios
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added unit tests that prove my fix is effective or that my feature works
### Documentation
- [ ] I have made corresponding changes to the documentation as follows:
- - [ ] New feature (non-breaking change which adds functionality): I have created the New Feature page in the project wiki explaining the functionality and how to use it. If relevant, I have committed sample data to the core seed to have usable examples in GardenWorld.
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected): I have documented the change in a clear way that everyone in the community can understand the impact of the change.
- - [ ] Improvement (improves and existing functionality): This documentation is needed if the improvement changes the way the user interacts with the system or the outcome of a process/task changes. If it is just, for instance, a performance improvement, documentation might not be needed. 
- [ ] The changed/added documentation is in the project wiki (not privately-hosted pdf files or links pointing to a company website) and is complete and self-explanatory.

### CODE REVIEW

File: org.adempiere.base/src/org/compiere/model/MColumn.java [hunk 1]
```diff
@@ -750,7 +748,21 @@ public String getConstraint(String tableName)
...
 	}	//	getConstraint
-	
+
+	/**
+	 * Get a correct Unique UU index name
+	 * @param columnName
+	 * @param indexName
+	 * @return validated indexName
+	 */
+	public static String validateUuIdxIndex(String columnName, String indexName) {
```
Comment (CarlosRuiz-globalqss) [inline org.adempiere.base/src/org/compiere/model/MColumn.java:758]: a better name for this method is maybe getUUIDIndexName.

Also, I think the method must just receive the tableName as parameter.

**Review by CarlosRuiz-globalqss — CHANGES_REQUESTED** [review]

File: org.adempiere.base/src/org/compiere/model/MColumn.java (hunk 1 above)
Comment (nmicoud) [reply org.adempiere.base/src/org/compiere/model/MColumn.java:758]: Hi,
I can change the method name and its parameter, but in this case, I should also move it to MTable or MTableIndex?
wdyt?

Comment (nmicoud) [reply org.adempiere.base/src/org/compiere/model/MColumn.java]: Done

**Comment by CarlosRuiz-globalqss** [comment]
@nmicoud / @hengsin - please check comment at https://idempiere.atlassian.net/browse/IDEMPIERE-6064?focusedCommentId=50641

**Comment by CarlosRuiz-globalqss** [comment]
replaced by https://github.com/idempiere/idempiere/pull/2290

**Pull request closed without merging**